### ISSUE TITLE
Add support for .NET 6.0

### ISSUE THREAD

**Issue by kevin-david** [rr-wfm/MSBuild.Sdk.SqlProj#197]
@jmezach mentioned in https://github.com/rr-wfm/MSBuild.Sdk.SqlProj/issues/165#issuecomment-875666253 that once .NET 6.0 has a go-live license (supported in production), support could be added to this SDK as well. 

6.0.0-rc1 was released on Sept 14th and has one: https://devblogs.microsoft.com/dotnet/announcing-net-6-release-candidate-1/

**Comment by jmezach** [comment]
You are absolutely right and we've already done some work to support .NET 6. In fact, PR #194 does exactly that. Unfortunately we've ran into an issue with DACFX (see Microsoft/DacFx#33) which doesn't seem to work on .NET 6 due to some breaking changes. Since that isn't open source there isn't really a lot I can do about that, except making sure that those that can make a difference are aware of the issue.

**Comment by DennisPlaydon** [comment]
Hey Jonathon. I noticed that the [preview version of DacFx](https://github.com/microsoft/DacFx/issues/33#issuecomment-961399413) is out and you mentioned that it fixed this bug. Is there any chance you could release a preview version of SqlProj which uses the preview version of DacFx?

**Comment by iremezoff** [comment]
Hi! I'm also following the case. What about to release preview/beta version with .net6 supporting depending on preview DacFx-version?

**Comment by jmezach** [comment]
I have a preview version in the works in #194 that will add support for .NET 6. Just waiting for a review.

**Comment by iremezoff** [comment]
Is it going to publish a preview version on nuget once the PR is merged?

**Comment by jmezach** [comment]
That's the idea yes.

**Comment by jmezach** [comment]
I have just uploaded version 2.0.0-beta.1 to NuGet and should be available shortly, which includes support for .NET 6.

**Comment by alexmg** [comment]
@jmezach I am using the `2.0.0-beta.1` package with the .NET 6.0 SDK and it seems to be working as before. Thank you for your efforts getting this beta update across the line. Hopefully the DacFx release will not be too far down the road.

**Comment by jmezach** [comment]
@alexmg Thanks for trying it out and the positive feedback. Much appreciated. From what I understand the DacFx team is planning to release before the end of the year, so hopefully soon.

**Comment by RPagels** [comment]
@jmezach, I am also using the 2.0.0-beta.1 package with the .NET 6.0 SDK and works great! Thank you!  Is there a timeframe for the non-beta version to be released?

**Comment by jmezach** [comment]
See my previous comment. Still waiting for a stable release of DacFx.

**Comment by adeal** [comment]
@jmezach it looks like a stable build was finally released! [160.5400.1](https://www.nuget.org/packages/Microsoft.SqlServer.DacFx/160.5400.1).
